herd_behaviour.nlogo

```diff
@@ -27,37 +29,87 @@ to setup
     set v-dir pseudo-random-vector
     set color blue
     set w 0
+    set g-dir [0 0]
   ]
   let number_informed_members (number_herd_members * ( 0.01 * (percent_informed)))
   ask n-of number_informed_members turtles [
     set w  weight
     set color red
+    set g-dir [125 125]
   ]
 end
 
 to go
   ask turtles[
-    ifelse count other turtles in-radius avoidance_range > 0 [
+    let base [0 0] ;; base of function calculated in every
```

Comment:
hab function vergessen... diese funktion, die wir festgelegt haben, die in jeder gleichung gerechnet werden muss.